### PR TITLE
fixed linting errors

### PR DESCRIPTION
**Pull request by romeokienzler**

### CODE REVIEW

File: analyze/spark-ts-trends.ipynb [hunk 1]
```diff
@@ -46,13 +46,13 @@
...
    "outputs": [],
    "source": [
-    "from pyspark import SparkContext, SparkConf\n",
-    "from pyspark.sql import SparkSession\n",
-    "import os\n",
-    "import pickle\n",
-    "import numpy as np\n",
-    "import shutil\n",
-    "import pandas as pd"
+    "from pyspark import SparkContext, SparkConf  # noqa\n",
```
Comment (ptitzler) [inline analyze/spark-ts-trends.ipynb:49]: What's the purpose of all these `#noqa`?

File: analyze/spark-ts-trends.ipynb [hunk 2]
```diff
@@ -144,31 +144,31 @@
...
-    "                              lon_lat[index][1]]=[coefficients[1]]\n",
+    "                              lon_lat[index][1]] = [coefficients[1]]\n",
     "                pickle.dump(lon_lat_slope, open(\n",
-    "                    data_dir+\"lon_lat_slope.p\", \"wb\"))\n",
+    "                    data_dir + \"lon_lat_slope.p\", \"wb\"))\n",
     "            except ValueError:\n",
     "                print(\"Could not convert data\")\n",
-    "            except:\n",
-    "                print('skipping, Generic Error')  # TODO remove bare except\n",
+    "            except:  # noqa -TODO remove bare except\n",
```
Comment (ptitzler) [inline analyze/spark-ts-trends.ipynb:170]: Why `noqa`?

**Comment by ptitzler** [comment]
`# noqa` is in most cases not the appropriate solution to resolving linting errors.

File: analyze/spark-ts-trends.ipynb (hunk 1 above)
Comment (romeokienzler) [reply analyze/spark-ts-trends.ipynb:49]: I need to have imports in later in the file as I have to import wget first before I can import stuff I get with wget

File: analyze/spark-ts-trends.ipynb (hunk 2 above)
Comment (romeokienzler) [reply analyze/spark-ts-trends.ipynb:170]: I need to catch this exception explicitly - but can't tell which one to catch without re-running the flow which I currently can't - will fix this once I anyway need to rerun the flow

Comment (ptitzler) [reply analyze/spark-ts-trends.ipynb:170]: Why not use https://docs.python.org/3/library/exceptions.html#Exception. It's the catch all

**Comment by romeokienzler** [comment]
> `# noqa` is in most cases not the appropriate solution to resolving linting errors.

we could deactivate the import order rule in this repo and I use noqa only for that single exception todo until fixed?

**Comment by ptitzler** [comment]
> we could deactivate the import order rule in this repo and I use noqa only for that single exception todo until fixed?

If the error indicates that the import order is not correct, why not change the order?

**Comment by romeokienzler** [comment]
> > we could deactivate the import order rule in this repo and I use noqa only for that single exception todo until fixed?
> 
> If the error indicates that the import order is not correct, why not change the order?

because I can't - I import wget, then download a library, then use another import to actually import the downloaded library

**Comment by romeokienzler** [comment]
> > `# noqa` is in most cases not the appropriate solution to resolving linting errors.
> 
> we could deactivate the import order rule in this repo and I use noqa only for that single exception todo until fixed?

nevermind - the import rule isn't active in GH - only locally, so I've just removed all "noqa" hints in this PR

**Review by ptitzler — APPROVED** [review]
LGTM; visual inspection only + confirmed that linting issues appear to be properly addressed

**Comment by ptitzler** [comment]
@lresende?

**Pull request merged**